Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -912,6 +913,8 @@ instance noMaxOrder : NoMaxOrder Ordinal :=
 instance succOrder : SuccOrder Ordinal.{u} :=
   SuccOrder.ofSuccLeIff (fun o => o + 1) succ_le_iff'
 
+instance succAddOrder : SuccAddOrder Ordinal := ⟨fun _ => rfl⟩
```

Comment:
```suggestion
instance instSuccAddOrder : SuccAddOrder Ordinal := ⟨fun _ => rfl⟩
```

Reply:
Should the `succOrder` instance just above get renamed too?

Reply:
Yes, please.

Reply:
Otherwise LGTM. Thanks!
bors d+